BP/history/characters/finnish.txt

```diff
@@ -95,49 +95,6 @@
 	}
 }
 
-109703 = {
-	name = "PA_iviO_"
-	dynasty = 100388
-	religion = "finnish_pagan"
-	culture = "finnish"
-	440.1.1 = {
-		birth = yes
-	}
-	510.1.1 = {
-		death = yes
-	}
-}
-
-1301194 = {
-	#fictional
-	name = "Vesa"
-	dynasty = 100388
-	religion = "finnish_pagan"
-	culture = "finnish"
-	father = 109703
-	490.1.1 = {
-		birth = yes
-	}
-	560.1.1 = {
-		death = yes
-	}
-}
-
-1301195 = {
-	#fictional
-	name = "Mielitty"
-	dynasty = 100388
-	religion = "finnish_pagan"
-	culture = "finnish"
-	father = 1301194
-	540.1.1 = {
-		birth = yes
-	}
-	610.1.1 = {
-		death = yes
-	}
-}
-
```

Comment:
No longer any title associated?

Reply:
Most of those were moved to _lt_rascek_changed_characters.txt because specific aspects about them changed to reflect the new cultures/religions; for those chars, they became Karelians given the county cultures.